lib/controller.js

```diff
@@ -557,6 +557,29 @@ controller.provisionTessel = function(opts) {
     });
 };
 
+
+controller.unbrickTessel = function(opts) {
+  if (!opts.openwrt) {
+    logs.info('You must use the --openwrt option to proceed with unbricking.');
+    return Promise.reject();
```

Comment:
In order to be consistent with existing error handing, I would recommend calling `Promise.reject` with the info string you are printing out just before. So replace the above two lines with:
```
return Promise.reject('You must use the --openwrt option to proceed with unbricking.');
```
Because a string is returned instead of an `Error`, the CLI will print out a WARN instead of ERR. I'm not particularly pleased with the way the CLI has different error handling based on return type but we can clean that up with one comprehensive PR in the future.